terraform/cloudfoundry/acl_bbs.tf

```diff
@@ -0,0 +1,10 @@
+resource "aws_network_acl_rule" "60_bbs" {
+    network_acl_id = "${aws_network_acl.cf.id}"
+    protocol = "tcp"
+    rule_number = 60
+    rule_action = "allow"
+    from_port = 8889
+    to_port = 8889
+    cidr_block = "10.0.32.0/20"
```

Comment:
Can you move all references to `10.0.32.0/20` to a named variable such as `cell_cidr_all`? It'll make it easier to understand the meaning of the value and reduce the chance of us making a typo.